pkg/api/kptfile/v1/types.go

```diff
@@ -268,6 +268,15 @@ type Function struct {
 	//	image: set-labels
 	Image string `yaml:"image,omitempty" json:"image,omitempty"`
 
+	// Exec specifies the function binary executable.
```

Comment:
Why don't we make it executable + args like `kpt fn eval --exec`?
If so, we can remove field `ExecArgs`.

Reply:
Actually I need to clarify. `Exec` is `executable + args` (see the tests), I am using ExecArgs for internal purposes (note the `-` tags for json and yaml). I will take another stab at cleaning it up to remove the confusion.

Reply:
Closing the loop on this. Did another pass today and was able to remove `ExecArgs` from kptfile definition. `exec` field supports exec-with-args as we do with `fn eval`.

Thanks for pointing out.